tests/RobotFramework/tests/cumulocity/Operation/c8y_Command_Fails

```diff
@@ -0,0 +1,5 @@
+[exec]
```

Comment:
The content of `c8y_Command_Fails` and `c8y_Command_Success` are the same. Can't you just have one file?

Reply:
There is a `forceful_timeout` that was missing. I added it.